src/drivers/imx/interrupt.c

```diff
@@ -340,9 +347,9 @@ static inline void irq_handler(void *data, uint32_t line_index)
 
 		if (!--tries) {
 			tries = IRQ_MAX_TRIES;
-			trace_irq_error("irq_handler(): IRQ storm, status "
-					PRIx64,
-					get_irqsteer_interrupts(line_index));
+			tr_err(&irq_i_tr, "irq_handler(): IRQ storm, status "
+			       PRIx64,
+			       get_irqsteer_interrupts(line_index));
```

Comment:
Does PRIx64 actually work? When I initially put it in I wasn't aware that it didn't (due to how the parameters are passed in traces), is there any change to support 64-bit parameters to traces? If not, I think we need to split it in two words here and print them separately.

Reply:
Afaik there is no support for that and may cause surprising issues since va_ iterator moves to the high 32bits when reading next parameter (caught in perf params that had been converted before).

Reply:
Yeah, I was seeing zeros even though the value is nonzero. Can you change this instance for me so that I'm seeing both halves of what get_irqsteer_interrupts() returned? Thanks.